packages/typst/build.sh

```diff
@@ -2,16 +2,26 @@ TERMUX_PKG_HOMEPAGE=https://typst.app/
 TERMUX_PKG_DESCRIPTION="A new markup-based typesetting system that is powerful and easy to learn"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.10.0"
+TERMUX_PKG_VERSION="0.11.0-rc1"
```

Comment:
I think it is better to wait for a stable release version...

Reply:
Emmm.. Normally, a stable version will be tagged as `Latest Release`, release candidate version will be tagged as `Pre-Release`, so no additional checks are required. I don't know why rc version is shipped as Release rather than Pre-Release in typst...

Reply:
ok, I shall wait for stable version and update this PR.

Reply:
I have updated the version to latest stable 0.11.0.